galaxy/templates/_helpers.tpl

```diff
@@ -161,7 +161,7 @@ cp -anL /galaxy/server/config/sanitize_allowlist.txt /galaxy/server/config/mutab
 cp -anL /galaxy/server/config/data_manager_conf.xml.sample /galaxy/server/config/mutable/shed_data_manager_conf.xml;
 cp -anL /galaxy/server/config/tool_data_table_conf.xml.sample /galaxy/server/config/mutable/shed_tool_data_table_conf.xml;
 cp -aruL /galaxy/server/tool-data {{.Values.persistence.mountPath}}/;
-cp -aruL /galaxy/server/tools {{.Values.persistence.mountPath}}/tools | true;
+cp -aruL /galaxy/server/tools {{.Values.persistence.mountPath}}/ | true;
```

Comment:
Should this also be `|| true` ?

```suggestion
cp -aruL /galaxy/server/tools {{.Values.persistence.mountPath}}/ || true;
```

Reply:
Good catch. Do we want/need that on the other cp commands?

Reply:
I guess it wouldn't hurt but honestly I don't even remember what scenario we were accommodating with this one .